protocol/agents.json

```diff
@@ -365,6 +365,11 @@
       "versioned": true,
       "type": "wrapper"
     },
+    {
+      "identifier": "chat-ui-kit",
```

Comment:
I would suggest `chat-react-ui-components` to match the repo name and ensure we don't have a clash later if we introduce new kits for Kotlin/Swift

Reply:
Yeah